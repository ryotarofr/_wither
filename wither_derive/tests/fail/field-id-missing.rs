use serde::{Serialize, Deserialize};
use wither::Model;

#[derive(Serialize, Deserialize, Model)]
struct BadModel {
    name: String,
}

fn main() {}
