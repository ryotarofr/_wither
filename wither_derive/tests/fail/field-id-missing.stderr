error: wither models must have a field `id` of type `Option<bson::oid::ObjectId>`
 --> $DIR/field-id-missing.rs:5:8
  |
5 | struct BadModel {
  |        ^^^^^^^^
